app/src/main/java/org/simple/clinic/scheduleappointment/ScheduleAppointmentSheetController.kt

```diff
@@ -119,9 +123,17 @@ class ScheduleAppointmentSheetController @Inject constructor(
   }
 
   private fun scheduleDefaultAppointmentDateForSheetCreates(events: Observable<UiEvent>): Observable<PotentialAppointmentDate> {
-    return events
-        .ofType<ScheduleAppointmentSheetCreated>()
-        .withLatestFrom(configProvider) { _, config -> config.defaultTimeToAppointment }
+    val protocolStream = currentFacilityStream()
+        .map { Optional.toOptional(it.protocolUuid) }
```

Comment:
We also have an extension function `toOptional()` that can change this to look like:

```kotlin
.map { it.protocolUuid.toOptional() }
```